src/components/Graph/Graph.js

```diff
@@ -4,53 +4,104 @@ import PropTypes from 'prop-types';
 import GraphVis from 'react-graph-vis';
 
 const characteristics = {
```

Comment:
Right now, the path for visiting this page is `/home/characteristics`. Do we want to change it to something else? `/home/birds`?